src/Model/Table/LazyTableTrait.php

```diff
@@ -45,7 +45,7 @@ public function ensureTables(array $fixtures)
             if ($class === false) {
                 throw new \RuntimeException("Unknown fixture '$name'.");
             }
-            $fixture = new $class();
+            $fixture = new $class($this->connection()->configName());
```

Comment:
I tried this fix and it doesn't seem to help. [TestFixture::__construct()](https://github.com/cakephp/cakephp/blob/master/src/TestSuite/Fixture/TestFixture.php#L95) doesn't take any arguments.